src/index.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
It is nice to see you're using Tailwind CSS for styling. I'm a big fan of Tailwind too. I use it all the time.

- [ ] But, it would be nice if you could change the font from the default one tailwind brings to Lato font, the one in the original design specification. You can get Google's Lato font and add the style to the body to have Lato globally.

![image](https://user-images.githubusercontent.com/50941074/194719937-be3054ce-711e-4497-af16-ef2e8ce5708a.png)